wpilibc/src/main/native/cpp/SPI.cpp

```diff
@@ -165,8 +165,6 @@ SPI::SPI(Port port) : m_port(static_cast<HAL_SPIPort>(port)) {
              static_cast<uint8_t>(port) + 1);
 }
 
-SPI::~SPI() = default;
```

Comment:
The destructor needs to be defined somewhere if it’s declared in the header—did you mean to remove it from the header as well?

Reply:
oops, nope